pytorch_lightning/trainer/trainer.py

```diff
@@ -570,6 +570,7 @@ def __init__(
         self.__init_profiler(profiler)
 
         # init logger flags
+        self.logger: Optional[LightningLoggerBase]
```

Comment:
nit: do you think it'd be easier to move this out of the logger connector into the trainer? the same way `self.__init_profiler` is run above? 

https://github.com/PyTorchLightning/pytorch-lightning/blob/4ee01b715c543b7db969bb4422273cc07449d522/pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py#L80-L93

Reply:
Yes, we could totally do this. The LoggerConnector is nowadays more of a "result orchestrator". The setting of attributes part could be moved from the connector to the trainer as it has been done for the other connectors.

Not for this PR though, as we want to include the fix in the bug-fix branch